src/main/java/org/kohsuke/github/GHWorkflowRun.java

```diff
@@ -285,8 +285,9 @@ public void rerun() throws IOException {
     /**
      * Downloads the logs.
      * <p>
-     * The logs are in the form of a zip archive. The full log file is at the root and called {@code 1_build.txt}. Split
-     * log files are also available in the {@code build} directory.
```

Comment:
This information was incorrect so I removed it. The structure of the log is more complex than that and depends on job names.